### DISCUSSION TITLE
Date range with two inputs with custom overlay

### DISCUSSION THREAD

**Discussion by corrot** [gpbl/react-day-picker#1142] [Support]
I'm using react-day-picker and I want to enable an overlay to add some custom components inside date range picker with two inputs.

There are both examples(range with two inputs and overlay) but combination of these two doesn't meet my expectations.

Here is a sandbox: https://codesandbox.io/s/react-day-picker-base-q7li8

Desired behavior is just like in this example, just with an overlay: http://react-day-picker.js.org/examples/input-from-to

**Comment by salacoste** [comment]
You need to use additional wrapper component as property of InputDatePicker component and inside your wrapper you could use your custom content and casual DatePicker option.

**Comment by corrot** [comment]
Can you please show me an example? I can not see anything like InputDatePicker in documentation
